File: 1835.py
class Solution:
    def getXORSum(self, arr1: List[int], arr2: List[int]) -> int:
        res1, res2 = 0, 0
        for n in arr1:
            res1 ^= n
        for n in arr2:
            res2 ^= n
        return res1 & res2

File: 1835.cpp
struct Solution {
    int getXORSum(const vector<int>& arr1, const vector<int>& arr2) const {
        int res1 = 0, res2 = 0;
        for (const int &n: arr1){
            res1 ^= n;
        }
        for (const int &n: arr2){
            res2 ^= n;
        }
        return res1 & res2;
    }
};
